modules/virt-about-hpp-operator.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-architecture.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-hpp-operator_{context}"]
+= About hostpath-provisioner-operator
+
+The `hostpath-provisioner-operator` deploys and manages the multi-node hostpath-provisioner (HPP) and related resources.
+
+image::cnv_components_hpp-operator.png[hpp-operator components]
+
+.hostpath-provisioner-operator components
+[cols="1,1"]
+|===
+|*Component* |*Description*
+
+|`deployment/hpp-pool-hpp-csi-pvc-block-<worker_node_name>`
+|Provides a worker for each node where hostpath provisioner (HPP) is designated to run. The pods mount the specified backing storage to the node.
```

Comment:
Very small nit. I think the normal usage is "mount x on y", so "The pods mount the specified backing storage on the node."